assemble-and-restart

```diff
@@ -13,6 +13,23 @@ else
     /usr/libexec/s2i/assemble
 fi
 
+### 
+# Check "ODO_S2I_DEPLOYMENT_DIR" environment variable and if it's present,
+# copy content of directory to /opt/app-root/backup directory
+# Ref: https://github.com/redhat-developer/odo/issues/445
+
+if [ ! -d /opt/app-root/backup ]; then
+    mkdir  /opt/app-root/backup
+fi
+
+if [ -z "$ODO_S2I_DEPLOYMENT_DIR" ]
+then
+   echo "\$ODO_S2I_DEPLOYMENT_DIR is empty"
```

Comment:
I don't think that user will need to see this message. It won't tell him/her anything useful.